src/node.cc

```diff
@@ -1657,14 +1658,14 @@ void SetupProcessObject(Environment* env,
 
 
 void SignalExit(int signo) {
-  uv_tty_reset_mode();
```

Comment:
It was arguably a bug to call `uv_tty_reset_mode()` first because the freebsd workaround right below it needs to happen as quickly as possible.